gulpfile.js

```diff
@@ -152,7 +152,7 @@ function createWebpackConfig(defines, output) {
           loader: 'babel-loader',
           exclude: /src\/core\/(glyphlist|unicode)/, // babel is too slow
           options: {
-            presets: pdfjsNext ? undefined : ['es2015'],
+            presets: pdfjsNext ? undefined : ['env'],
```

Comment:
The same change must be done at https://github.com/timvandermeij/pdf.js/blob/98b9d5ec37398bcd1eeb784f2b220f57bb9cabb8/gulpfile.js#L1002 as well.

Reply:
Done in the new commit.